src/app/shell/info/about/about.component.html

```diff
@@ -1,20 +1,22 @@
-<div *ngIf="(platformInformation$ | async)?.companyInformationItems.length; else noInfo">
-  <div class="info-content">
-    <div>
-      <h2 class="info-title">{{ (platformInformation$ | async).title }}</h2>
-    </div>
-    <div class="info-text">
-      <div *ngFor="let item of (platformInformation$ | async)?.companyInformationItems">
-        <p><strong>{{ item.sectionName }}</strong></p>
-        <p>{{ item.description }}</p>
+<ng-container *ngIf="!(isLoading$ | async)">
+  <div *ngIf="(platformInformation$ | async)?.companyInformationItems.length; else noInfo">
+    <div class="info-content">
+      <div>
+        <h2 class="info-title">{{ (platformInformation$ | async).title }}</h2>
+      </div>
+      <div class="info-text">
+        <div *ngFor="let item of (platformInformation$ | async)?.companyInformationItems">
+          <p><strong>{{ item.sectionName }}</strong></p>
+          <p>{{ item.description }}</p>
+        </div>
+        <p>
+          Ваші пропозиції по покращенню роботи сервісу можна надсилати:<a href="mailto:pozashkillia@gmail.com"
+            target="_blank"><span class="info-link"> Pozashkillia@gmail.com</span></a>
+        </p>
       </div>
```

Comment:
this static sections also should be removed

Reply:
removed